user-guide/Advanced_Functionality/Swarming/EnableSwarming.md

```diff
@@ -0,0 +1,102 @@
+
+## Enabling Swarming
+
+When you have made sure prerequisites are met, and you are ready to enable Swarming, you can do so using the SLNet message *EnableSwarmingRequest* (with the default parameters) in SLNetClientTest tool or an Automation script. Enabling Swarming via DataMiner Cube is currently not yet possible.
+
+> [!IMPORTANT]
+> When you enable swarming, the element configuration will be moved from disk to database. This change can only partially be rolled back. See [Partially rolling back Swarming](xref:SwarmingRollback). We highly recommend taking a backup before enabling Swarming.
+
+To enable Swarming using SLNetClientTest tool:
+
+1. [Connect to the DMA using the SLNetClientTest tool](xref:Connecting_to_a_DMA_with_the_SLNetClientTest_tool).
+
+   > [!CAUTION]
+   > Always be extremely careful when using this tool, as it can have far-reaching consequences on the functionality of your DataMiner System.
+
+1. Go to the *Build Message* tab of the main window of the SLNetCLientTest tool.
+
+1. In the *Message Type* drop-down list, select the message *Skyline.DataMiner.Net.Swarming.EnableSwarmingRequest*.
+
+   This message will need to be sent with its default settings, so do not make any changes to the settings for the message.
+
+1. Click *Send Message*.
+
+   The prerequisites will be checked for all Agents in the system, and when they are met, Swarming will enabled and all Agents in the DMS will be **restarted**.
+
+   The prerequisites check can take some time, as checking the usage of legacy alarm IDs in scripts can take a while.
+
+   On first startup, the existing element XML files will be moved from disk into the database. This can also take several minutes. While this is happening, a message will be displayed on any clients that are trying to connect.
+
+> [!NOTE]
+> The migrated element files will be **temporarily** backed up in the *Recycle Bin* (e.g. *2024_11_20 11_03_12_300_ElementFolder_BeforeSwarmingMigration.zip*). We recommend that you store these files somewhere safe if you ever want to access these again later or if you want to be able to [partially roll back Swarming](wref:SwarmingRollback).
+
+## Verifying that the feature is active
+
+Once all Agents have been restarted, you can verify whether the Swarming feature has been activated by checking if the following line is present in the *SLSwarming.txt* log file:
+
+`SwarmingManager::SwarmingManager|INF|-1|Swarming enabled`
+
+You can find this log file in DataMiner Cube via *System Center* > *Logging* > *dataminer* > *Swarming*. The log file also shows details on the migration of elements that happened during DMA startup.
+
+Once Swarming is enabled, you can go to *System Center* > *Agents* > *Status* > *Swarm* to swarm elements for which this feature is supported.
```

Comment:
Note: only in clusters. Single agents do not show this button, even if swarming is enabled